test/NuGet.Tests.Apex/NuGet.PackageManagement.UI.TestContract/ApexTestUIProject.cs

```diff
@@ -95,6 +95,11 @@ public bool VerifyDeprecatedPackageOnTopOfInstalledTab()
             return result?.IsPackageDeprecated == true;
         }
 
+        public PackageItemViewModel GetPackageItemOnInstalledTabById(string packageId)
+        {
+            return _packageManagerControl.PackageList.PackageItems.Where(x => x.Id == packageId).FirstOrDefault();
```

Comment:
Probably best to have this method return the package list so you can account for nulls.